_posts/2021-06-30-gsoc'21-projects.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "JabRef GSoC’21 Projects"
+id: gsoc'21-projects
+author: JabRef GSoC Mentee
+bg: jabref-font
+color: white
+---
+
+Hey everyone!. Today's blog is about the projects selected for GSoC and a little sneak peek of the projects.
+
+## Microsoft Word Integration
+
+We are working on a Word add-in that allows users to select and insert citations directly from the Word document. The JabRef Word Add-in can be used without having your reference manager open or even installed. Once you sign in to JabRef Word Addin, your JabRef library is downloaded from the cloud, and you can continue working on your document. JabRef word addin opens as a separate panel in Word alongside your document window. So your whole document remains in your view as you search, select and insert references.
+
+![in-3](https://user-images.githubusercontent.com/62339705/123989792-db4e9d80-d9e6-11eb-8f1b-0527ce94774d.gif)
```

Comment:
Please do not link an external file, because when moving away from github, this might get lost. Could you please upload it into this git repo? @mohit038